config/config.py.template

```diff
@@ -500,9 +500,15 @@ MLLIB_DECISION_TREE_TEST_OPTS = MLLIB_COMMON_OPTS + [
 ]
 
 if MLLIB_SPARK_VERSION >= 1.2:
+    ensembleTypes = ["mllib.RandomForest"]
+    if MLLIB_SPARK_VERSION >= 1.3:
+        ensembleTypes.append("mllib.GradientBoostedTrees")
```

Comment:
The API used by the perf tests (`GradientBoostedTrees`) came in [1.3](https://github.com/apache/spark/commit/15cacc81240eed8834b4730c5c6dc3238f003465#diff-7b5c1db0b1926a36f418b53fcf807db0); it was called (`GradientBoosting`) in 1.2 so these perf tests won't work there.